packages/@react-spectrum/s2/.gitignore

```diff
@@ -0,0 +1,2 @@
+icons
+page.css
```

Comment:
Does this need dist or style/dist? They are part of the Makefile clean.

Reply:
the root `.gitignore` in the repo takes care of all dist dirs already